### ISSUE TITLE
[Feature Request]  Add alternative html/template HTMX support to Templ

### ISSUE THREAD

**Issue by unknovvn** [Melkeydev/go-blueprint#268]
### Tell us about your feature request

There are multiple projects which use go-htmx stack, but not all of them are using Templ for templating. It would be nice if standart html/template alternative would be provided for this stack.

### Disclaimer

- [X] I agree

**Comment by briancbarrow** [comment]
Thanks for submitting the issue @unknovvn. Are you thinking we should split the HTMX option into two separate options? 

Something like this?

- HTMX
- Template

**Comment by unknovvn** [comment]
Hey @briancbarrow. At the moment there is only HTMX with Templ blueprint available, so it looks like people are not interested in blueprints without using HTMX (for example Templ without HTMX). IMHO HTMX option could be split into two:
- HTMX and Templ
- HTMX and std html/template

**Comment by Ujstor** [comment]
It is easy to remove the HTMX component from the generated project and use a templ with html, cutom css and js in asset dir.

Generated project is not final solution, just a starting point

**Comment by unknovvn** [comment]
@Ujstor I totally agree with you. Although it does not cover using HTMX with std html/template case

**Comment by newtoallofthis123** [comment]
Would be kinda cool to have!
